Cabal/doc/developing-packages.rst

```diff
@@ -1716,10 +1716,16 @@ system-dependent values for these fields.
     Cabal can make sure that specified programs are built and on the ``PATH`` before building the component in question.
     It will always do so for internal dependencies, and also do so for external dependencies when using Nix-style local builds.
 
+    :pkg-field:`build-tool-depends` was added in Cabal 2.0, and it will
```

Comment:
nit nit: Should we start doing the 1 sentance per line thing? :) I did that with the documentation I wrote.